.github/workflows/lint.yml

```diff
@@ -0,0 +1,40 @@
+name: golangci-lint
+on: [pull_request]
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  pull-requests: read
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.19'
```

Comment:
can you use  Go 1.20 version as we already build the nats-manager with 1.20?

Reply:
this go version is just required for the linter. it is not the one we use to build the software.